Glade2d/Graphics/Renderer.cs

```diff
@@ -12,10 +12,12 @@ namespace Glade2d.Graphics
 {
     public class Renderer : MicroGraphics
     {
+        private int _width, _height;
+        
         readonly Dictionary<string, IPixelBuffer> textures = new Dictionary<string, IPixelBuffer>();
         public Color BackgroundColor { get; set; } = Color.Black;
         public Color TransparentColor { get; set; } = Color.Magenta;
-        public bool ShowPerf { get; set; } = false;
+        public bool ShowPerf { get; set; } = true;
```

Comment:
I'm not sure changing the renderer default to `true` is an intentional part of this PR?

Reply:
The more I think about this, the more having `true` as a default makes sense. But turning it off should be discoverable. I probably should just address this with a good doc about the renderer.

Reply:
We resolved this via slack convo: we'll leave this on for now as we have quite a bit of perf work and can change it later.